sdk/spring/spring-cloud-azure-feature-management-web/src/main/java/com/azure/spring/cloud/feature/manager/FeatureManagerSnapshot.java

```diff
@@ -44,4 +44,24 @@ public Mono<Boolean> isEnabledAsync(String feature) {
 
         return featureManager.isEnabledAsync(feature).doOnSuccess((enabled) -> requestMap.put(feature, enabled));
     }
+
+    /**
+     * Checks to see if the feature is enabled. If enabled it check each filter, once a single filter returns true it
```

Comment:
> If enabled it check each filter, once a single filter returns true ...

Seems this content is related to current method. How about delete it?

Reply:
I am not sure what you mean by this.

Reply:
np, ignore my previous comment.